File: prueba.R
# tiempoInicio <- Sys.time()
# prepararAmbiente <- function(){
#   if (!is.element("tidyverse", installed.packages())){
#     print("Yes")
#   }
#   if (!is.element("readxl", installed.packages())){
#     print("Yes")
#   }
# }
# 
# prepararAmbiente()

library("tidyverse")
library("readxl")

args = commandArgs(trailingOnly=TRUE)
# path_loc <- "D:/Datos de Usuario/rgamezv/Downloads/prueba"
path_loc <- args[1]
archivoOrigen <- args[2]
# archivoOrigen <- "telecom.csv"
setwd(path_loc)

dataOriginal <- read_csv(archivoOrigen)
resultado <- slice(dataOriginal, 1)

# path_loc <- "D:/Datos de Usuario/rgamezv/Downloads/resultadoPrueba"
path_loc <- args[3]
setwd(path_loc)
nombreArchivoFinal <- str_glue("resultado.csv")
write_excel_csv(resultado, nombreArchivoFinal)
# tiempoFin <- Sys.time()
# 
# print(tiempoFin - tiempoInicio)

File: prueba.py
import sys, subprocess

locCSV = sys.argv[1]
nombreCSV = sys.argv[2]
locResultado = sys.argv[3]

locCSV = "D:/Datos de Usuario/rgamezv/Downloads/prueba"
nombreCSV = "telecom.csv"
locResultado = "D:/Datos de Usuario/rgamezv/Downloads/resultadoPrueba"

comandoR = 'Rscript "D:/Datos de Usuario/rgamezv/Downloads/prueba/prueba.R" "{}" "{}" "{}"'.format(locCSV, nombreCSV, locResultado)

def main():
    print(comandoR)
    subprocess.run(comandoR)

if __name__ == '__main__':
    main()